.idea/compiler.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="11" />
+    <bytecodeTargetLevel target="1.8" />
```

Comment:
I'd rather you don't commit these files, but keep them locally for your IDE

Reply:
Sure, so do I need to add it to .gitignore?

Reply:
No, revert the changes, commit and push